x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/queries/WeightedTokensQueryBuilderTests.java

```diff
@@ -361,21 +361,21 @@ public void testIllegalValues() {
         {
             IllegalArgumentException e = expectThrows(
                 IllegalArgumentException.class,
-                () -> new WeightedTokensQueryBuilder("field name", weightedTokens, new TokenPruningConfig(-1f, 0.0f, false))
+                () -> new WeightedTokensQueryBuilder("field name", weightedTokens, new TokenPruningConfig(-1, 0.0f, false))
             );
-            assertEquals("[tokens_freq_ratio_threshold] must be between [1.0] and [100.0], got -1.0", e.getMessage());
+            assertEquals("[tokens_freq_ratio_threshold] must be between [1] and [100.0], got -1", e.getMessage());
```

Comment:
I think you need to change the number format due to the [casting](https://github.com/elastic/elasticsearch/pull/103663/files#diff-b695b29ec072d11fbc936715d5755705d33745cda2bacc59650d5382fdc62b8aR51).
```suggestion
            assertEquals("[tokens_freq_ratio_threshold] must be between [1] and [100], got -1", e.getMessage());
```